FrontEnd/src/components/ProfilePage/FormComponents/GeneralInfo.js

```diff
@@ -62,7 +64,8 @@ const GeneralInfo = (props) => {
     const [logoImage, setLogoImage] = useState(props.profile.logo_image);
     const [bannerImageError, setBannerImageError] = useState(null);
     const [logoImageError, setLogoImageError] = useState(null);
-    const [identifierError, setIdentifierError] = useState(null);
+    const [edrpouFieldError, setEdrpouFieldError] = useState(null);
+    const [rnokppFieldError, setRnokppFieldError] = useState(null);
```

Comment:
Could we use `react-hook-form` here?

Reply:
We'll discuss it with the team and maybe try to implement this form using react-hook-form.